.github/workflows/docker.yml

```diff
@@ -55,16 +57,23 @@ jobs:
       - name: Prepare build args
         id: build_args
         run: |
-          github_sha_short=${SHA:0:7}
-          echo "IMAGE_TAG=sha-${github_sha_short}" >> $GITHUB_ENV
+          # Generate SHA image tag
+          github_sha_short=${GITHUB_SHA:0:7}
+          echo "IMAGE_SHA_TAG=sha-${github_sha_short}" >> $GITHUB_ENV
+
+          # Generate branch image tag
+          branch_name=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}
+          echo "IMAGE_BRANCH_TAG=branch-${branch_name}" >> $GITHUB_ENV
```

Comment:
slashes are okay in image names? this has the slash, right?

Reply:
Right; I'm sanitizing the name in the Makefile (since the Makefile target can be invoked from more than one place); let me try and see what it looks like if I do it in the HCL.